nuxt-app/package.json

```diff
@@ -22,6 +22,7 @@
     "@justeattakeaway/pie-cookie-banner": "0.26.3",
     "@justeattakeaway/pie-css": "0.12.1",
     "@justeattakeaway/pie-icons-webc": "0.25.1",
+    "@justeattakeaway/pie-lottie-player": "0.0.1",
```

Comment:
Does it still work when imported from `pie-webc`?

Reply:
Good point!